common/next/next.config.js

```diff
@@ -42,9 +45,10 @@ const createConfig =
                 source: '/account/:path*',
                 destination: `${identityHost}/account/:path*`,
               },
+              ...rewriteEntries,
             ];
           }
-          return [];
+          return [...rewriteEntries];
```

Comment:
nit: do we need to spread here, or can we just return `rewriteEntries` directly?